utils/resultwriter.go

```diff
@@ -121,6 +129,57 @@ func (rw *ResultsWriter) PrintScanResults() error {
 	}
 	return nil
 }
+
+func AppendVulnsToJson(results *Results) error {
```

Comment:
As mentioned. if you are not using the results writer. move this to your enrich command.

Reply:
fixed